simple_message/src/messages/robot_status_message.cpp

```diff
@@ -66,6 +66,7 @@ bool RobotStatusMessage::init(industrial::simple_message::SimpleMessage & msg)
   bool rtn = false;
   ByteArray data = msg.getData();
   this->init();
+  this->setCommType(msg.getCommType());

```

Comment:
This is more of a style issue.  Using accessors in a class function is not necessary (although fine) since these classes are relatively simple.  Not required to fix, just my opinion.